EdFi.Ods.Utilities.Migration/logistics/scripts/modules/build-edfi-standard-descriptors.psm1

```diff
@@ -0,0 +1,148 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+$ErrorActionPreference = "Stop"
+$repositoryNames = @('EdFi.Ods.Utilities.Migration')
+& "$PSScriptRoot\..\..\..\logistics\scripts\modules\load-path-resolver.ps1"  $repositoryNames 
+Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics\scripts\modules\tasks\TaskHelper.psm1')
+Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics\scripts\modules\packaging\packaging.psm1')
+
+function New-EdFiStandardDescriptorsPackage {
+    param(
+        [string]
+        $OutputPath = (Join-Path $env:temp 'EdFi.Standard.Descriptors/'),
+
+        # NOTE: the target folder version is the api version the migration utility is migrating to NOT the data standard version
+        [hashtable[]]
+        $PackageFiles = @(
+            @{ src = (Get-RepositoryResolvedPath "../../Ed-Fi-Standard/v3.1/Descriptors/"); target = (Join-Path $OutputPath "any/netcoreapp3.1/Descriptors/3.1") }
+        )
+    )
+
+    Write-Host "OutputPath $OutputPath"
+    $PackageFiles | % { Write-HashtableInfo $_ }
+
+    $config = @{
+        OutputPath = $OutputPath
+        PackageFiles = $PackageFiles
+    }
+
+    $script:tasks = @(
+        'Remove-TempDirectory'
+        'New-TempDirectory'
+        'Copy-PackageFiles'
+        'New-PackageNuspec'
+    )
+
+    $script:result = @()
+
+    $elapsed = Use-StopWatch {
+        foreach ($task in $tasks) {
+            $script:result += Invoke-Task -name $task -task { & $task $config }
+        }
+    }
+
+    Test-Error
+
+    $script:result += New-TaskResult -name '-' -duration '-'
+    $script:result += New-TaskResult -name $MyInvocation.MyCommand.Name -duration $elapsed.format
+
+    return $script:result | Format-Table
```

Comment:
should be able to remove the TaskHelper.psm1 specific functions here: `Write-HashtableInfo`, `Use-StopWatch`, `Invoke-Task`, `New-TaskResult` ect.. Some of this is nice to have when it's easily available but it's not necessary for the functionality of this script.

Reply:
These functions are required for build-edfi-standard-descriptors.psm1